src/components/Select.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { SelectProps } from 'rendition';
+import styled, { StyledFunction, withTheme } from 'styled-components';
+import asRendition from '../asRendition';
+import { radius } from '../theme';
+import { px } from '../utils';
+
+const Base = (styled.select as StyledFunction<
+	{ emphasized?: boolean } & React.HTMLProps<HTMLSelectElement>
+>)`
+  border-radius: ${px(radius)};
+  height: ${props =>
+		px(props.emphasized ? props.theme.space[5] : props.theme.space[4])};
+  font-size: inherit;
+  border: 1px solid ${props => props.theme.colors.gray.main};
+  padding-top: 0px;
+  padding-bottom: 0px;
+  padding-left: 16px;
+  padding-right: ${props =>
+		px(props.emphasized ? props.theme.space[5] : props.theme.space[4])};
+  background-color: white;
+  -webkit-appearance: none;
```

Comment:
I know we can't use something like [autoprefixer](https://github.com/postcss/autoprefixer) but it would be great to drop the vendor prefixes somehow.

Reply:
`styled-components` auto prefixes by default, so I can just remove these